pipelines/pro_image.go

```diff
@@ -0,0 +1,62 @@
+package pipelines
+
+import (
+	"context"
+	"fmt"
+
+	"dagger.io/dagger"
+	"github.com/grafana/grafana-build/containers"
+)
+
+func ProImage(ctx context.Context, dc *dagger.Client, directory *dagger.Directory, args PipelineArgs) error {
+	debianPackageFile := dc.Host().Directory("./").File(args.ProImageOpts.Deb)
+
+	hostedGrafanaRepo, err := containers.CloneWithGitHubToken(dc, args.ProImageOpts.GithubToken, "https://github.com/grafana/hosted-grafana", "main")
+	if err != nil {
+		return fmt.Errorf("cloning hosted-grafana repo: %w", err)
+	}
+
+	socketPath := "/var/run/docker.sock"
+	socket := dc.Host().UnixSocket(socketPath)
+
+	container := dc.Container().From("golang:1.20-alpine").
+		WithUnixSocket(socketPath, socket).
+		WithExec([]string{"apk", "add", "--update", "make", "docker", "git", "jq", "jsonnet"}).
```

Comment:
I think we can actually remove every dependency here except `docker`.

Reply:
done